### ISSUE TITLE
Logs a warning once per day: event of type UNKNOWN not handled: {"type":"hello"}

### ISSUE THREAD

**Issue by BastianVoigt** [Ullink/simple-slack-api#251]
The simple-slack-api logs a warning once per day:

    full_message: event of type UNKNOWN not handled: {"type":"hello"}
    level: 4
    level_name: WARN
    logger_name: com.ullink.slack.simpleslackapi.impl.SlackWebSocketSessionImpl
    message: event of type UNKNOWN not handled: {"type":"hello"}

**Comment by eximius313** [comment]
I'm also suffering from this one.
@BastianVoigt  - could you do PR for this library?

**Comment by eximius313** [comment]
BTW. Simmilar to this one I got when somebody joined the channel:
```
14:45:02.856 [warn] com.ullink.slack.simpleslackapi.impl.SlackWebSocketSessionImpl - event of type UNKNOWN not handled: {"type":"member_joined_channel","user":"XX","channel":"XX","channel_type":"G","team":"XX","inviter":"XX","event_ts":"XX","ts":"XX"}
```

**Comment by BastianVoigt** [comment]
@eximius313 I did file a PR, but no response:

https://github.com/Ullink/simple-slack-api/pull/252

**Comment by eximius313** [comment]
I see...
Are you handling `member_joined_channel` and `group_history_changed` in your fork?

**Comment by BastianVoigt** [comment]
No, just the hello world message. Others can be added similarly
